client/src/Visit.test.tsx

```diff
@@ -101,6 +125,17 @@ describe('Visit', () => {
       }
     );
 
+    const getChatThreadIdFromTeamsLinkSpy = jest.spyOn(GetMeetingLink, 'getChatThreadIdFromTeamsLink');
```

Comment:
Same here, move this test case into `TeamsMeeting.test.tsx` since this only applies to `TeamsMeeting.tsx`